lib/active_set/sorting/operation.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require_relative '../attribute_instruction'
+
+class ActiveSet
+  module Sorting
+    class Operation
+      def initialize(set, instructions_hash)
+        @set = set
+        @instructions_hash = instructions_hash
+      end
+
+      def execute
```

Comment:
The logic for this method is to check whether all of the instructions appear to be processable by the `ActiveRecordStrategy`, and if they are to attempt to sort using the `ActiveRecordStrategy` (with all of the instructions, as you can't split sorting instructions). We then double check that all of the instructions were indeed successfully processed by the `ActiveRecordStrategy`, and if they were, we return that result. Otherwise (if either some instructions don't appear to be processable by ActiveRecord or some instructions weren't processed by ActiveRecord), we sort with the `EnumerableStrategy`.